tests/test_frontdoor.py

```diff
@@ -165,7 +165,7 @@ def test_happy_flow():
     referral.delete()
 
 
-def _do_happy_flow(supplier="Foxglove"):
+def _do_happy_flow(supplier="EON"):
```

Comment:
Guessing this change fixes some unhappy tests?

Reply:
Yeah - Foxglove is commented out in supplier options :)